### DISCUSSION TITLE
Pull latest YouTube video transcript

### DISCUSSION THREAD

**Discussion by jlin3** [jdepoix/youtube-transcript-api#249] [Ideas]
Hello! First off, thank you so much for creating this super helpful API!

I am working on a project to pull the transcripts of the latest video upload for a list of YouTube channels.

Can you help advise how I could do this?

Go thru list of usernames, sort their video ids by date uploaded descending, and pull transcripts for those videos?

Thanks,
J

**Comment by jdepoix** [comment]
Hi! I feel like most of what you are trying to do here (other than fetching the transcript) is beyond the scope of this module. Maybe there's some people around here who have some advice for you anyways, but I feel like you'd have to do most of that with the official YouTube API and then use this module to get the actual transcript.
